src/partials/Media/Media.tsx

```diff
@@ -22,12 +23,16 @@ class Media extends React.Component<MediaProps, MediaState> {
   }
 
   render() {
-    const { type, data } = this.props;
+    const { type, data, classes } = this.props;
 
     switch (type) {
       case 'image':
         return <img src={this.getImgUrl(data)} alt={data && data.alt ? data.alt : ''} className={'mediaImage'} />;
         break;
+      case 'background-image':
```

Comment:
I think the style of the background image should be handle outside this componend just in CSS.

Reply:
maybe use camel case for props